src/main/java/com/uid2/shared/store/ScopedStoreReader.java

```diff
@@ -55,8 +64,11 @@ private long loadContent(String path) throws Exception {
         try (InputStream inputStream = this.contentStreamProvider.download(path)) {
             ParsingResult<T> parsed = parser.deserialize(inputStream);
             latestSnapshot.set(parsed.getData());
-            LOGGER.info(String.format("Loaded %d %s", parsed.getCount(), dataTypeName));
-            return parsed.getCount();
+
+            final int count = parsed.getCount();
+            latestEntryCount.set(count);
```

Comment:
What happens in case of errors? If we don't have an active snapshot, would we be left with a previous count that could mask an issue?

Reply:
I see RotatingStoreVerticle has `consecutive_refresh_failures`. But what if we had a metric "last_successful_refresh" and an alert when it's too old?

Reply:
setting to -1 on failure

Reply:
Decided to leave this for now. `getSnapshot()` will return whatever the last successful `loadContent()` retrieved, so the metric matches that. We can do refresh failure count as another ticket.